README.md

```diff
@@ -25,23 +24,53 @@ https://insightsoftwareconsortium.github.io/ITKDoxygen/
 
 To create the the documentation locally, run the following commands:
 
+### 1. Clone the ITKDoxygen Repository
+
+### 2. Build the Docker Image
 ```shell
-docker build --build-arg TAG=v1.1.0 . -f Dockerfile -t itk-doxygen
-docker build -f Dockerfile -t itk-doxygen .
+docker build . -f Dockerfile -t itk-doxygen
+```
 
-export TAG=v1.2.0
-docker run --env TAG --name itk-dox itk-doxygen
+### 3. Export TAG Environment Variable
+The `TAG` environment variable is used to specify a filename for the tarballs to be copied from the Docker container to your local working directory (see step 5). Useful for naming multiple versions of the documentation. Here we have set it to *alpha*, but you may change it as needed.
```

Comment:
Maybe you can create a cross-reference to step 5 when mentioning it.

Reply:
Sounds good, added!